packages/editor/src/editor/canvas/data-class/DataClassDialog.tsx

```diff
@@ -26,26 +26,36 @@ import { rowToCreateData, variableTreeData } from '../../../data/variable-tree-d
 import { flexRender, getCoreRowModel, getFilteredRowModel, useReactTable, type ColumnDef, type Row } from '@tanstack/react-table';
 import { createInitForm } from '../../../data/data';
 
-export const DataClassDialog = () => (
+export const DataClassDialog = ({
+  text = 'Create from data',
+  iconVariant = 'outline',
+  className,
+  worfkflowButtonsInit = true
+}: {
+  text?: string;
+  iconVariant?: 'outline' | 'primary' | undefined;
+  className?: string;
+  worfkflowButtonsInit?: boolean;
+}) => (
   <Dialog>
     <DialogTrigger asChild>
-      <Button icon={IvyIcons.DatabaseLink} variant='outline'>
-        Create from data
+      <Button icon={IvyIcons.DatabaseLink} variant={iconVariant} size='large' className={className}>
+        {text}
       </Button>
     </DialogTrigger>
```

Comment:
If I where you, instead of customize all this props I would simply add the trigger button as prop

```tsx
export const DataClassDialog = ({ children, worfkflowButtonsInit = true }: { children: ReactNode; worfkflowButtonsInit?: boolean }) => (
  <Dialog>
    <DialogTrigger asChild>{children}</DialogTrigger>
    <DialogContent>
      <DialogHeader>
        <DialogTitle>Create from data</DialogTitle>
      </DialogHeader>
      <DataClassSelect worfkflowButtonsInit={worfkflowButtonsInit} />
    </DialogContent>
  </Dialog>
);
```

and then use it just like that
```tsx
<PaletteElement text='Data'>
  <DataClassDialog worfkflowButtonsInit={false}>
    <Button icon={IvyIcons.DatabaseLink} size='large' />
  </DataClassDialog>
</PaletteElement>
```

and
```tsx 
<DataClassDialog>
  <Button icon={IvyIcons.DatabaseLink} size='large' variant='outline'>
    Create from data
  </Button>
</DataClassDialog>
```